services/motion/explore/explore.go

```diff
@@ -0,0 +1,575 @@
+// Package explore implements a motion service for exploration. This motion service model is a temporary model
```

Comment:
This should go in the builtin directory, since it is also builtin (rdk:builtin:explore).

Reply:
Spoke offline with both Ray and Tess and we will be keeping it in its own folder until full intergation into builtin, due to a certain level of redundancy of helper functions which requires a more thorough integration than what we plan here.